### DISCUSSION TITLE
can't acces selected-rows-label in QTable

### DISCUSSION THREAD

**Discussion by liusvel90** [quasarframework/quasar#10178] [General - Components / Directives / etc] [answered]
Hi all
I want to display a custom message and I cannot access the selected row
I attach a resource with the code I am testing
[https://codepen.io/liusvel/pen/jOmaxyG?editors=1001](url)
thanks in advance
quasar rocks
;)

**Comment by MilosPaunovic** [comment, marked as answer]
If you use single selection (like in the provided example), you can use:

```js
getSelectedString () {
  return `User selected ${selected.value[0].name}`    
}
```

In case of multiple selection, map values and extract names or whichever property you need.

**Reply by liusvel90** [reply]
Thanks for your time
I left some hair on my keyboard looking for the solution
I leave the code in case someone might need it
